src/highdicom/sr/templates.py

```diff
@@ -3582,21 +3582,21 @@ def get_volumetric_roi_measurement_groups(
 
         Parameters
         ----------
-        tracking_uid: str, optional
+        tracking_uid: Union[str, None], optional
             Unique tracking identifier
-        finding_type: Union[highdicom.sr.CodedConcept, pydicom.sr.coding.Code], optional
+        finding_type: Union[highdicom.sr.CodedConcept, pydicom.sr.coding.Code, None], optional
             Finding
-        finding_site: Union[highdicom.sr.CodedConcept, pydicom.sr.coding.Code], optional
+        finding_site: Union[highdicom.sr.CodedConcept, pydicom.sr.coding.Code, None], optional
             Finding site
-        reference_type: Union[highdicom.sr.GraphicTypeValues, highdicom.sr.GraphicTypeValues3D], optional
+        reference_type: Union[highdicom.sr.GraphicTypeValues, highdicom.sr.GraphicTypeValues3D, None], optional
```

Comment:
This was actually wrong

```suggestion
        reference_type: Union[highdicom.sr.CodedConcept, pydicom.sr.coding.Code, None], optional
```